app/brave_main_delegate_browsertest.cc

```diff
@@ -84,6 +85,7 @@ IN_PROC_BROWSER_TEST_F(BraveMainDelegateBrowserTest, DisabledFeatures) {
   const base::Feature* disabled_features[] = {
 #if !defined(OS_ANDROID)
     &apps::kAppDiscoveryRemoteUrlSearch,
+    &translate::kTFLiteLanguageDetectionEnabled,
```

Comment:
cc @atuchin-m

Reply:
We have discussed this.
https://github.com/brave/brave-browser/issues/19098